opcua/server/address_space.py

```diff
@@ -180,34 +180,46 @@ def add_nodes(self, addnodeitems, user=User.Admin):
     def _add_node(self, item, user):
         result = ua.AddNodesResult()
 
-        if item.RequestedNewNodeId in self._aspace:
+        # Check if the null NodeId is given
+        if item.RequestedNewNodeId.is_null():
+            self.logger.debug("RequestedNewNodeId is null")
+            if item.BrowseName.NamespaceIndex:
```

Comment:
this works with tests and works with my server/client applications....

Using the NamespaceIndex of the BrowserName was the only solution to get custom NamespaceIndex to the server since the standard definition... see comment down